clients/gtest/gemm_gtest.cpp

```diff
@@ -3,7135 +3,135 @@
  * ************************************************************************ */
```

Comment:
This file is practically rewritten, to be data-driven instead of having test cases statically compiled into C++.